src/client/components/TopBar.tsx

```diff
@@ -42,12 +42,15 @@ export const TopBarFiller = styled.div`
 const Container = styled.div`
   width: 100%;
   height: 100%;
-  max-width: 80rem;
-  padding: 0 2rem !important;
```

Comment:
@palmenhq was there a reason for adding `!important`? Couldn't see that it was overridden when removing it :)

Reply:
🤷  if i remember correctly embark had some overrides but maybe they have been removed or idk. If it looks fine anyways i guess its fine anyways :D

Reply:
Cool! I tried it in embark and it looked fine, will double check once it's out in staging 🙌